index.html

```diff
@@ -2,10 +2,112 @@
+    <h1>Start using your big software instead of learning how to deploy it</h1>
+    <p>Conjure-up lets you summon up a big-software stack as a &ldquo;spell&rdquo; &mdash; a model of the stack, combined with extra know-how to get you from an installed stack to a fully usable one.</p>
+    </div>
+  </div>
+</div>
 
-<p>Deployments so easy it's almost magical</p>
+<div class="p-strip--dark">
+  <div class="row row--get-started">
+    <div class="col-12">
+      <h2 class="row--get-started__title">Get started</h2>
+      <ul class="p-list p-list--get-started">
+        <li class="p-list__item">$ sudo snap install conjure-up classic</li>
+        <li class="p-list__item">$ conjure-up</li>
+      </ul>
+    </div>
+  </div>
+</div>
 
-<h2>Conjure-up is a power tool for getting started with big software</h2>
+<div class="p-strip p-strip--conjure-up">
+  <div class="row">
+    <h2>Conjure-up spells</h2>
 
-<p>Conjure-up lets you summon up a big-software stack as a &ldquo;spell&rdquo; - a model of the stack, combined with extra know-how to get you from an installed stack to a fully usable one. Start using your big software instead of learning how to deploy it.</p>
+    <div class="col-12 theme__outline u-equal-height">
+      <div class="col-4 p-card--highlighted">
+        <div class="p-heading-icon">
+          <div class="p-heading-icon__header">
+            <img class="p-heading-icon__img" src="{{ site.asset_path }}dba11aee-kubernetes.svg" alt="Kubernetes logo" />
+            <h3 class="p-heading-icon__title">Kubernetes</h3>
+          </div>
+          <p><code>$ conjure-up kubernetes</code></p>
+          <p>Get the Canonical Distribution of Kubernetes deployed and running all on a single machine. </p>
+          <p><a class="p-link--external" href="https://www.ubuntu.com/containers/kubernetes#anchored">Kubernetes with conjure-up</a></p>
+        </div>
+      </div>
+
+      <div class="col-4 p-card--highlighted">
+        <div class="p-heading-icon">
+          <div class="p-heading-icon__header">
+            <img class="p-heading-icon__img" src="{{ site.asset_path }}873bc93e-OpenStack_Logo_Mark.svg" alt="Openstack logo" />
+            <h3 class="p-heading-icon__title">Openstack</h3>
+          </div>
+          <p><code>$ conjure-up Openstack</code></p>
+          <p>Easily deploy real-world OpenStack on a single machine using LXD containers.</p>
+          <p><a class="p-link--external" href="https://www.ubuntu.com/download/cloud/conjure-up">Openstack with conjure-up</a></p>
+        </div>
+      </div>
+
+      <div class="col-4 p-card--highlighted">
+        <h3>More spells</h3>
+        <p><code>$ conjure-up</code></p>
+        <p>View all spells by using the command above, or view them in the GitHub repository.</p>
+        <p><a class="p-link--external" href="https://github.com/conjure-up/spells">Go to spell registry</a></p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="p-strip--light p-strip--conjure-help">
```

Comment:
Created a proposal for a white strip to help here in the future: https://github.com/vanilla-framework/vanilla-framework/issues/966